<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload_file"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionUpload_file_2"/>
    <addaction name="actionShow_History"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHistory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpload_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionUpload_file_2">
   <property name="text">
    <string>Open history file</string>
   </property>
  </action>
  <action name="actionShow_History">
   <property name="text">
    <string>Show History</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
